packages/api/src/external/ehr/athenahealth/shared.ts

```diff
@@ -32,7 +32,7 @@ export function createMetriportAddresses(patient: PatientResource): Address[] {
       addressLine1: address.line[0] as string,
       addressLine2: address.line.length > 1 ? address.line.slice(1).join(" ") : undefined,
       city: address.city,
-      state: normalizeState(address.state),
+      state: normalizeUSStateForAddress(address.state),
```

Comment:
@thomasyopes this is throwing a `BadRequestError` (400) whereas before it was throwing `Error` (500).

Reply:
Generally it's not true that normalization in only for requests, but having it default to 400 and overriding with a 500 is fine (if I need it here). I can use the safe version and throw a new error type if undefined.

Reply:
> Generally it's not true that normalization in only for requests

Yeah, `BadRequestError` _can be_ translated to a HTTP status code 400, but it represents bad input and so can be used throughout the codebase, not just for HTTP requests.

Just to confirm: from your perspective, it's ok to make this change for Athena then?

Reply:
Yea, I just need to throw an error 👍